src/pdn/src/PdnGen.tcl

```diff
@@ -399,12 +399,19 @@ proc check_design_state {} {
 
 proc check_orientations {orientations} {
   set valid_orientations {R0 R90 R180 R270 MX MY MXR90 MYR90}
+  set lef_orientations {N R0 FN MX S R180 FS MY E R90 FE MYR90 W R270 FW MXR90}
```

Comment:
I think these should be
FN -> MY
FS -> MX
E -> R270
W -> R90

Reply:
![image](https://user-images.githubusercontent.com/761514/128397006-19d941dc-83b4-4382-87f7-57bae1798da2.png)

Reply:
fixed